src/app/pages/ArticlePage/index.stories.jsx

```diff
@@ -144,16 +141,15 @@ export const Burmese = props => (
 );
 
 export const ArticlePageWithRelatedContent = props => (
```

Comment:
Should we now delete the props if they are not being used?

Reply:
Yea I'll remove the props that aren't needed.